### DISCUSSION TITLE
Best way to deal with data not reaching the bottom of formations

### DISCUSSION THREAD

**Discussion by lucianomarquetto** [gempy-project/gempy#800] [Q&A]
Hi all! 

In the region I am working on I have several borehole logs or vertical electrical soundings that do not reach the bottom of some formations, but that I think would be important to add because some of those formations (deeper ones) have few datapoints. I saw a similar question in https://github.com/cgre-aachen/gempy/issues/551 but did not completely understand how this inversion should be done or how should I alter my series order; or even if this is the best approach to my situation. 

To explain further: I have four main series with varied thicknessess as in the image below:

![image](https://github.com/cgre-aachen/gempy/assets/103533248/0dc0e7f2-67d9-4dee-ab47-53b6cf7f532b)
![image](https://github.com/cgre-aachen/gempy/assets/103533248/0a09abbd-4ae5-4131-b094-5e6f1b29bd9b)

Let's say in one point Unit1 goes down to -10m, and my borehole goes all the way down to -100 m but the bottom of Unit2 is not reached. One option would be to consider the bottom of Unit2 as -100 m even though it is not; another would be to add a surface point for Unit1 at -10m but not for Unit2. In this case, I would be inclined to do the former, as probably Unit2 would be at its final depths considering geological data available.

But then another situation where I have Unit3, which has only a few points in the dataset: Unit3 top is at -70m; the borehole bottom is at -80m but it did not reach the bottom of Unit3. Again I can decide between the two options as before (considering -80m as Unit3 bottom or leaving Unit3 off this datapoint). But in this case if I do the former, Unit3 thickness might be really wrong as the model will consider I have a real thickness and will squeeze the layer; but if I leave this point off, the model might understand there is no Unit3 in this area (as it happened in the left end of the image I uploaded, Unit3 in yellow).

So, in this case, what you think would be the best approach? I understand there are uncertainties in each method we chose, and in case there is no good answer to this question it is still interesting to know!

Cheers,

Luciano

Ohh and forgot: in case I need to 'invert' the layer, if I have an horizontal layer with azimuth, dip and polarity as 0,0,1, what I need to do is change polarity to -1 is that correct? Thanks!

**Comment by AlexanderJuestel** [comment]
Hello @lucianomarquetto,

I have some thoughts here: 

Borehole with Unit 1 at -10 m and terminating in Unit 2 without reaching the base of Unit 2: Definitely adding an interface point for Unit 1 but none for Unit 2 for now. 

What you could do with your Units 2 and 3 is using probabilistic modeling and Bayesian Interference. Please also see this publication here: https://publications.rwth-aachen.de/record/672787/files/de%20la%20Varga%20and%20Wellmann%202016_final.pdf

From Geological observations, you should know the thickness of Units 2 and 3 and can come up with a distribution of thicknesses (i.e. normal distribution with a mean of 100 m and a stdv of for instance 20 m). You could then draw random values from this distribution and add them in different model realizations to your model setup as Z locations for Unit 2 or 3.
